### PR TITLE
skip 3.5 on windows ci build wheel

### PR DESCRIPTION
**Pull request by ajfriend**
Playing around with this issue: https://github.com/joerick/cibuildwheel/issues/350

### CODE REVIEW

**Comment by henryiii** [comment]
Next experiment: can you set `CIBW_DEPENDENCY_VERSIONS` to `latest` instead of the default (`pinned`)? Not likely to be the problem, but might be work a test.

**Comment by ajfriend** [comment]
Weirdly, the workflows aren't getting triggered now...

**Comment by henryiii** [comment]
Github had an outage, maybe it was during that time?

**Comment by henryiii** [comment]
You could bump to cibuildwheel 1.4.2, and see if that triggers a rebuild too.

**Comment by henryiii** [comment]
Looks like it is trying to install dependencies:

```
  Installing build dependencies: started
  Installing build dependencies: still running...
  Installing build dependencies: finished with status 'done'
```

But it's still not finding them in the environment. Can you please set `CIBW_BUILD_VERBOSITY` to 1? That should show the search and links found for the PEP 517 setup. It's *really* verbose, and I which it was quieter, but at least we could see what it's finding and installing.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -1,48 +1,48 @@
-name: tests
```
Comment (qa8302) [inline .github/workflows/tests.yml:1]: -

Comment (ajfriend) [reply .github/workflows/tests.yml:1]: ?

**Comment by ajfriend** [comment]
Follow this issue: https://github.com/uber/h3-py/issues/138

**Pull request closed without merging**